modules/usertracking-impl/src/main/java/org/opencastproject/usertracking/impl/UserActionImpl.java

```diff
@@ -53,28 +54,33 @@
  */
 @Entity(name = "UserAction")
 @Access(AccessType.FIELD)
-@Table(name = "oc_user_action")
+@Table(name = "oc_user_action", indexes = {
+    @Index(name = "IX_oc_user_action_created", columnList = "created"),
+    @Index(name = "IX_oc_user_action_inpoint", columnList = "inpoint"),
+    @Index(name = "IX_oc_user_action_outpoint", columnList = "outpoint"),
+    @Index(name = "IX_oc_user_action_mediapackage_id", columnList = "mediapackage"),
```

Comment:
The field `mediapackage` is generated as `LONGTEXT` instead of `VARCHAR(128)`
Therefore the generation of the index throws the following exception:
`Internal Exception: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: BLOB/TEXT column 'mediapackage' used in key specification without a key length
Error Code: 1170`

Reply:
Fixed.